src/main/java/com/esri/samples/scene/get_elevation_at_a_point/GetElevationAtAPointSample.java

```diff
@@ -0,0 +1,168 @@
+import com.esri.arcgisruntime.mapping.Surface;
+import com.esri.arcgisruntime.mapping.view.Camera;
+import com.esri.arcgisruntime.mapping.view.Graphic;
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay;
+import com.esri.arcgisruntime.mapping.view.LayerSceneProperties;
+import com.esri.arcgisruntime.mapping.view.SceneView;
+import com.esri.arcgisruntime.symbology.TextSymbol;
+import com.esri.arcgisruntime.symbology.SimpleLineSymbol;
+
+public class GetElevationAtAPointSample extends Application {
+
+  private SceneView sceneView;
+  private GraphicsOverlay graphicsOverlay;
+
+  @Override
+  public void start(Stage stage) {
+
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene fxScene = new Scene(stackPane);
+      fxScene.getStylesheets().add(getClass().getResource("/css/style.css").toExternalForm());
+
+      // set title, size, and add JavaFX scene to stage
+      stage.setTitle("Get Elevation At A Point Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(fxScene);
+      stage.show();
+
+      // create a scene and add a basemap to it
+      ArcGISScene scene = new ArcGISScene();
+      scene.setBasemap(Basemap.createImagery());
+
+      // add the SceneView to the stack pane
+      sceneView = new SceneView();
+      sceneView.setArcGISScene(scene);
+      stackPane.getChildren().add(sceneView);
+
+      // add base surface for elevation data
+      Surface surface = new Surface();
+      surface.getElevationSources().add(new ArcGISTiledElevationSource("http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"));
+      scene.setBaseSurface(surface);
+
+      // create a point symbol and graphic to mark where elevation is being measured
+      SimpleLineSymbol elevationLineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, 0xFFFF0000, 3.0f);
+      Graphic polylineGraphic = new Graphic();
+      polylineGraphic.setSymbol(elevationLineSymbol);
+
+      // create a text symbol and graphic to display the elevation of selected points
+      TextSymbol elevationTextSymbol = new TextSymbol(20, null, 0xFFFF0000, TextSymbol.HorizontalAlignment.CENTER, TextSymbol.VerticalAlignment.BOTTOM);
+      Graphic elevationTextGraphic = new Graphic();
+      elevationTextGraphic.setSymbol(elevationTextSymbol);
+
+      // create a graphics overlay
+      graphicsOverlay = new GraphicsOverlay(GraphicsOverlay.RenderingMode.DYNAMIC);
+      graphicsOverlay.getSceneProperties().setSurfacePlacement(LayerSceneProperties.SurfacePlacement.RELATIVE);
+      sceneView.getGraphicsOverlays().add(graphicsOverlay);
+
+      // add a camera and initial camera position
+      Camera camera = new Camera(28.42, 83.9, 10000.0, 10.0, 80.0, 0.0);
+      sceneView.setViewpointCamera(camera);
+
+      // create listener to handle clicked
+      sceneView.setOnMouseClicked(event -> {
+
```

Comment:
I think you might need an `event.isStillSincePress()` to avoid the on click being handled at the end of a pan.

Reply:
Thanks Colin, I've got it in at line 108 when checking that the point clicked on is on the scene surface, so should be covered for that?

Reply:
Yeah that's ok. I would just have expected it to be the first thing you did in the listener. If it is a click as the result of a drag then there's no point in doing anything else.

Reply:
Good spot - I'll move that to happen directly after the click event.